src/Checkout/Helpers/ReserveStock.php

```diff
@@ -222,8 +274,8 @@ private function get_query_for_reserved_stock( $product_id, $exclude_order_id =
 		 * @since 4.5.0
 		 * @param string $query            The query for getting reserved stock of a product.
 		 * @param int    $product_id       Product ID.
-		 * @param int    $exclude_order_id Order to exclude from the results.
+		 * @param int    $exclude_customer_id customer_id to exclude from the results.
 		 */
-		return apply_filters( 'woocommerce_query_for_reserved_stock', $query, $product_id, $exclude_order_id );
+		return apply_filters( 'woocommerce_query_for_reserved_stock', $query, $product_id, $exclude_customer_id );
```

Comment:
Since users implementing this filter will have been expected an order ID, we shouldn't change the existing parameter.

Reply:
Yes, I've restored the `$exclude_order_id` and added `$exclude_customer_id ` to the end of the filter.